src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -130,7 +130,7 @@ class Request
     public $headers;
 
     /**
-     * @var string|resource
+     * @var null|string|resource
```

Comment:
actually, this could also be false, as we have `false === $this->content` several times

Reply:
at 1st glance i was wondering why it's false-able when it's already null-able...
looks like way too many types are allowed